slides/workshop_rap_for_managers_basics.qmd

```diff
@@ -0,0 +1,343 @@
+:::
+
+
+# How to track your progress
+
+The [RAP self-assessment tool](https://rsconnect/rsc/publication-self-assessment/) allows statistics producers to assess themselves against the department's RAP standards.
+
+::: {.fragment .fade-up}
+Importantly for managers, you can monitor your team's progress towards achieving the RAP baseline by viewing 'RAP progress by area' and 'overview of latest responses'.
+:::
+
+::: {.fragment .fade-up}
+You should have entries for <span style="color:#f67400;">**every publication**</span> you manage, and these should be updated either;
+:::
+
+> -   **When you make updates to your processes** that change your responses or,
+> -   **once a year minimum** even if you've made no changes.
+
+::: {.fragment .fade-up}
+This is so we always have an accurate picture of RAP in the department to share with analytical leaders.
+:::
+
+# RAP Mythbusting
+
+- <span style="color:#f67400;">**Myth**</span>: **If we move to a RAP process, we'll be relying on computers rather than humans with experience**
+- <span style="color:#f67400;">**Reality**</span>: RAP is about using computers to do the things they're good at (speeding up repetitive processes), so humans can do the things they're good at (critical thinking and quality assurance based on knowledge and context). It's about making the best use of both. 
+
+<br>
+
+- <span style="color:#f67400;">**Myth**</span>: **Once our project has achieved the RAP standards, it'll be perfect**
+- <span style="color:#f67400;">**Reality**</span>: RAP is a set of best practice standards, but there are always ways to improve code efficiency. You should be looking to make continuous improvements to your processes where necessary. 
+
+
+# More RAP Mythbusting
+
+- <span style="color:#f67400;">**Myth**</span>: **If we move to RAP best practice, we'll have to rewrite all our SQL code into R**
+- <span style="color:#f67400;">**Reality**</span>: RAP is about using the best tool for the job. If you're using SQL and it's working for you, that's great, and there's no pressure to rewrite code if it's not necessary. We especially recommend using SQL for any heavy processing work with large datasets. 
+
+<br>
+
+- <span style="color:#f67400;">**Myth**</span>: **RAP is complicated and hugely time consuming**
+- <span style="color:#f67400;">**Reality**</span>: Gradual, small changes towards best practice all add up - you don't have to change everything all at once!
```

Comment:
could be nice and acknowledge they're probably already doing a lot of the stuff they need to